pkg/cloud/services/networking/trunk.go

```diff
@@ -77,6 +79,44 @@ func (s *Service) getOrCreateTrunkForPort(eventObject runtime.Object, port *port
 	return trunk, nil
 }
 
+func (s *Service) RemoveTrunkSubports(trunkID string) error {
+	mc := metrics.NewMetricPrometheusContext("trunk", "deletesubports")
```

Comment:
We really need a metrics strategy. I'd probably leave this out, tbh, but in absence of any related policy I'm not going to object.

Reply:
It looks to me like this is a duplicate. We are already getting the metrics in `networking.go` above.
Can you remove this @mquhuy ?

Reply:
Yes, sure. Coming up